src/opencv.mk

```diff
@@ -52,12 +54,12 @@ define $(PKG)_BUILD
     # openexr isn't available on x86_64-w64-mingw32
     # opencv builds it's own libIlmImf.a
     $(if $(findstring x86_64-w64-mingw32,$(TARGET)),\
-        $(SED) -i 's/OpenEXR//' '$(BUILD_DIR)/unix-install/opencv.pc')
-    $(SED) -i 's,share/OpenCV/3rdparty/,,g' '$(BUILD_DIR)/unix-install/opencv.pc'
-    $(INSTALL) -m755 '$(BUILD_DIR)/unix-install/opencv.pc' '$(PREFIX)/$(TARGET)/lib/pkgconfig'
-
+    $(SED) -i 's/OpenEXR//' '$(1).build/unix-install/opencv.pc')
+    $(SED) -i 's,share/OpenCV/3rdparty/,,g' '$(1).build/unix-install/opencv.pc'
+    $(INSTALL) -m755 '$(1).build/unix-install/opencv.pc' '$(PREFIX)/$(TARGET)/lib/pkgconfig'
+    
     '$(TARGET)-g++' \
-        -W -Wall -Werror -ansi -pedantic \
-        '$(1)/samples/c/fback_c.c' -o '$(PREFIX)/$(TARGET)/bin/test-opencv.exe' \
-        `'$(TARGET)-pkg-config' opencv --cflags --libs`
+      -v -W -Wall -Werror -ansi \
+      '$(1)/samples/cpp/fback.cpp' -o '$(PREFIX)/$(TARGET)/bin/test-opencv.exe' \
+      `echo \`i686-w64-mingw32.static-pkg-config opencv --cflags --libs | $(SED) -r 's/-llib([0-9a-zA-Z]+)/-l\1/g'\``
```

Comment:
You can use `$$()` syntax instead of `, because it can be nested without escaping:
```
$$($$())
```
(We replace `$` with `$$`, otherwise `make` would consider it its macro.)

Also, why do we need nested at all? Why not
```
`i686-w64-mingw32.static-pkg-config opencv --cflags --libs | $(SED) -r 's/-llib([0-9a-zA-Z]+)/-l\1/g'`
```
?

Reply:
That's a fair point, I guess.